metadata/org.apache.kafka/kafka-clients/3.5.1/reflect-config.json

```diff
@@ -139,6 +139,16 @@
       }
     ]
   },
+  {
+    "name": "org.apache.kafka.common.security.scram.internals.ScramSaslClient$ScramSaslClientFactory",
```

Comment:
Can we have a `typeReachable` condition here? This will include it for all projects that have kafka on the classpath.

Reply:
I'm pushing a couple of typeReachable conditions to both entries I've submitted. Let me know if they are ok or more strict ones are needed (in that case any help is appreciated).